src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/CostsService.java

```diff
@@ -1,39 +1,88 @@
 package uk.gov.companieshouse.overseasentitiesapi.service;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.companieshouse.api.model.payment.Cost;
 
 import java.util.Collections;
+import uk.gov.companieshouse.overseasentitiesapi.exception.SubmissionNotFoundException;
 
 @Service
 public class CostsService {
 
-    private static final String COST_DESC = "Register Overseas Entity fee";
+    private static final String REGISTER_COST_DESC = "Register Overseas Entity fee";
+
+    private static final String UPDATE_COST_DESC = "Update Overseas Entity fee";
+
     private static final String PAYMENT_ACCOUNT = "data-maintenance";
     private static final String RESOURCE_KIND = "overseas-entity";
-    private static final String PRODUCT_TYPE = "register-overseas-entity";
+    private static final String PRODUCT_TYPE_REGISTER = "register-overseas-entity";
+
+    private static final String PRODUCT_TYPE_UPDATE = "update-overseas-entity";
+
     private static final String CREDIT_CARD = "credit-card";
     private static final String DESCRIPTION_IDENTIFIER = "description-identifier";
     private static final String PAYMENT_SESSION = "payment-session#payment-session";
     private static final String KEY = "Key";
     private static final String VALUE = "Value";
 
     @Value("${OE01_COST}")
-    private String costAmount;
+    private String costRegisterAmount;
```

Comment:
Again here, register/update at the start of the names. Then immediately clear what journey they're used for.